sample-notificator-domain/src/main/groovy/life/qbic/business/notification/send/SendNotificationOutput.groovy

```diff
@@ -1,12 +0,0 @@
-package life.qbic.business.notification.send
-
-/**
- * <b><short description></b>
- *
- * <p><detailed description></p>
- *
- * @since 1.0.0
- */
-interface SendNotificationOutput {
-
-}
```

Comment:
Why did you delete that? this is the interface that is to be used to communicate to the email sending part in the app layer.

Reply:
And what will be the output here?
The use case itself is sending the email. This is one use case for the whole app for now. I think it can be added when it is required.

Reply:
no the use case must not send the email since this requires 3rd party tools